website/docs/d/is_placement_group.html.markdown

```diff
@@ -13,8 +13,8 @@ Provides a read-only data source for PlacementGroup. You can then reference the
 ## Example Usage
 
 ```hcl
-data "is_placement_group" "is_placement_group" {
-	id = "id"
+data "ibm_is_placement_group" "example" {
+  name = ibm_is_placement_group.example.name
```

Comment:
Existing is on the `Id`, this changes the argument to `name`, Is it correct ?

Reply:
yes. Referred from acceptance test

Reply:
Let's discuss.